packages/web3-shared/src/constants/index.ts

```diff
@@ -44,7 +44,11 @@ export const useTraderConstants = hookTransform(getTraderConstants)
 export const getTrendingConstants = transform(Trending)
 export const useTrendingConstants = hookTransform(getTrendingConstants)
 
-export const getRPCConstants = transformFromJSON(process.env.WEB3_CONSTANTS_RPC ?? '', RPC)
+let WEB3_CONSTANTS_RPC = ''
+try {
+    WEB3_CONSTANTS_RPC = process.env.WEB3_CONSTANTS_RPC ?? ''
+} catch {}
```

Comment:
why?

Reply:
`ReferenceError: "process" is undefined.`

it needs a special configuration on the webpack to support this constant.